docs/advanced_examples/utils/classifier_comparison_utils.py

```diff
@@ -21,30 +21,35 @@
 from sklearn.model_selection import train_test_split
 from sklearn.preprocessing import StandardScaler
 
+from concrete.ml.common.utils import get_model_name
 from concrete.ml.sklearn import DecisionTreeClassifier
 
 ALWAYS_USE_SIM = False
 
-# pylint: disable=too-many-locals,too-many-statements,too-many-branches
-def make_classifier_comparison(title, classifiers, decision_level, verbose=False):
+# pylint: disable=too-many-locals,too-many-statements,too-many-branches,invalid-name
+def make_classifier_comparison(
+    title, classifiers, decision_level, verbose=False, show_score=False, save_plot=False
+):
 
     h = 0.04  # Step size in the mesh
+    n_samples = 25 if get_model_name(classifiers[0][0]) == "KNeighborsClassifier" else 200
 
     X, y = make_classification(
-        n_samples=200,
+        n_samples=n_samples,
         n_features=2,
         n_redundant=0,
         n_informative=2,
         random_state=1,
         n_clusters_per_class=1,
     )
+    # pylint: disable-next=no-member
     rng = np.random.RandomState(2)
     X += 2 * rng.uniform(size=X.shape)
     linearly_separable = (X, y)
 
     datasets = [
-        make_moons(n_samples=200, noise=0.3, random_state=0),
-        make_circles(n_samples=200, noise=0.2, factor=0.5, random_state=1),
+        make_moons(n_samples=n_samples, noise=0.2, random_state=0),
```

Comment:
I reduced the noise especially for the knn experiments. 
The dataset is small, so these perturbations lead to outliers that perturb both the concrete and sklearn models.